src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,118 @@
+@import '../../index.css';
+
+/* Styling images */
+.logo{
+    width:3rem;
+    height:3rem;
+    margin-left:20px;
+}
+
+.burger-menu{
+    width:2.5rem;
+    height:2.5rem;
+}
+
+.name{
+    width: 10rem;
+    height:auto;
+}
+
+/* NavBar Styling */
+.navbar{
+    background-color:var(--light-gray);
+    position:fixed;
```

Comment:
removing this helps the 'hidden' content appear. it is a requirement for something else in the nav like the burger menu functionality?

Reply:
I'll play around with this Lindi thanks!